graph/patterns/change-tracking.md

```diff
@@ -0,0 +1,94 @@
+# Change Tracking
+
+Microsoft Graph API Design Pattern
+
+*The change tracking pattern provides the ability to keep API consumers in sync with changes in Microsoft Graph without having to continuously poll the API.*
+
+## Problem
+---------
+
+API consumers require an efficient way to keep data in sync with Microsoft Graph, and the API design should not allow for continuous polling as it'd be costly and wouldn't guarantee data integrity.
```

Comment:
I think this needs a rewrite, because many customers do poll against the delta api pretty frequently.
It's not idea, but we don't have that many webhooks to make it unnecessary.

SO some language more like 'api design should be optimized to avoid polling as it is costly for consumers and producers alike.'

Reply:
```suggestion
API consumers require an efficient way to keep data in sync with Microsoft Graph, and the API design should be optimized to avoid polling as it is costly for consumers and producers alike as well as wouldn't guarantee data integrity.
```

what do you think of this?